components/automate-cli/pkg/verifyserver/utils/httputils/httputil_test.go

```diff
@@ -8,42 +8,68 @@ import (
 )
 
 func TestMakeRequest(t *testing.T) {
-	// Create a test server to receive requests
-	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
-		// Check that the request body is correct
-		expectedBody := "{\"org\":\"chef\"}"
-		requestBody, err := ioutil.ReadAll(req.Body)
-		if err != nil {
-			t.Errorf("Unexpected error reading request body: %v", err)
-		}
-		if string(requestBody) != expectedBody {
-			t.Errorf("Unexpected request body. Expected %v, got %v", expectedBody, string(requestBody))
-		}
-		// Write a response to the client
-		rw.WriteHeader(http.StatusOK)
-		rw.Write([]byte("{\"status\":\"OK\"}"))
-	}))
-	defer server.Close()
-
-	// Call the Post function with a test URL and body
-	url := server.URL
-	body := map[string]string{"org": "chef"}
-	resp, err := MakeRequest(http.MethodPost, url, body)
-	if err != nil {
-		t.Errorf("Unexpected error from Post function: %v", err)
+	tests := []struct {
+		description         string
+		isInterfaceFunction bool
+	}{
+		{
+			description:         "Test make request function",
+			isInterfaceFunction: false,
+		},
+		{
+			description:         "Test make request interface function ",
+			isInterfaceFunction: true,
+		},
 	}
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
+			// Create a test server to receive requests
+			server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+				// Check that the request body is correct
+				expectedBody := "{\"org\":\"chef\"}"
+				requestBody, err := ioutil.ReadAll(req.Body)
```

Comment:
```suggestion
				requestBody, err := io.ReadAll(req.Body)
```

as ioutil.Read is deprecated , Also check all file your are editing if the are using this old deprecated function the update them

Reply:
Done